bool execution_success

# Total time the action primitive took to execute
float64 execution_time

# Total path length for the given trajectory
float64 path_length

# Maximum velocity encountered along the planned trajectory
float64 max_velocity